spec/suites/map/MapSpec.js

```diff
@@ -162,6 +162,31 @@ describe("Map", function () {
 		});
 	});
 
+	describe("#getBoundsZoom", function () {
+		var halfLength = 0.0001;
+		var bounds = [[-halfLength, -halfLength], [halfLength, halfLength]];
+		var padding = [100, 100];
+		var height = '400px';
+
+		it("returns high levels of zoom with small areas and big padding", function () {
+			var container = map.getContainer();
+			container.style.height = height;
+			document.body.appendChild(container);
+			expect(map.getBoundsZoom(bounds, false, padding)).to.be.equal(21);
+		});
+
+		it("returns multiples of zoomSnap when zoomSnap > 0 on any3d browsers", function () {
```

Comment:
Should be `it.skipInPhantom("returns ...")`. See `spec/suites/map/anim/Map.ZoomAnimationSpec.js` and the end of `spec/SpecHelper.js`